config/constants.yml

```diff
@@ -10,11 +10,15 @@
 TERMINUS_VERSION:       '0.12.0'
 
 # Connectivity
-TERMINUS_HOST:          'dashboard.pantheon.io'
+TERMINUS_HOST:          'terminus.pantheon.io'
 TERMINUS_PORT:          443
 TERMINUS_PROTOCOL:      'https'
 TERMINUS_SSH_HOST:      null
 
+# User Prompts
+DASHBOARD_HOST:         'dashboard.pantheon.io'
```

Comment:
for reference, I'm grepping through with `config.*host` as there seems to be 2 styles for referencing config vars:

```php
$config = Config::getAll();
$config['host']
```

And

```php
Config::get('host')
```

Not sure if there is a style preference for one over the other.